raiden/transfer/mediated_transfer/mediator.py

```diff
@@ -781,6 +1408,39 @@ def handle_block(state, state_change):
     return iteration
 
 
+def handle_block2(channelidentifiers_to_channels, state, state_change, block_number):
+    """ After Raiden learns about a new block this function must be called to
+    handle expiration of the hash time locks.
+    Args:
+        state (MediatorTransferState): The current state.
+    Return:
+        TransitionResult: The resulting iteration
+    """
+    close_events = events_for_close2(
+        channelidentifiers_to_channels,
+        state.transfers_pair,
+        block_number,
+    )
+
+    # Withdraw is handled by the channel once the close transaction is mined
+    # withdraw_events = events_for_withdraw(
```

Comment:
Is this superseeded by `events_for_close2`?

Reply:
no, the withdraw is handled by the channel, not the transfer task.